docs/batch-local-execution-mode.md

```diff
@@ -0,0 +1,90 @@
+# Design: Local execution mode for online serving queries
+
+## Background
+
+The query engine of risingwave is expected to support two kinds of query: highly concurrent point query and adhoc query.
+The characteristics of these two different kinds of queries are summarized as followings:
+
+|	|Point Query	|Adhoc query	|
+|---	|---	|---	|
+|Latency	|several ms	|ms to minutes	|
+|QPS	|1000 ~ 100000	|100 ~ 1000	|
+|SQL	|Simple	|Arbitrary complex	|
+|Result Set	|Small	|Small, Medium, Large	|
+|Use Case	|Dashboard	|Engineers, Data Scientist	|
+
+Our distributed query processing engine is designed for complex adhoc query, and it can’t meet the latency/qps 
+requirement of point query, and in this article we introduce local execution mode for point query.
```

Comment:
-> ... point queries. In this article, we'll introduce the local execution mode for point queries.